app/helpers/users_helper.rb

```diff
@@ -1,2 +1,15 @@
 module UsersHelper
+
+  def user_ud(user)
```

Comment:
this method name means nothing to the casual observer (me)  please rename it.

Also it's common to name methods that return true/false with a question mark, eg: `def is_empty?`